mwdb/model/migrations/versions/12849e43c922_new_favorites_table.py

```diff
@@ -0,0 +1,39 @@
+"""new_favorites_table
```

Comment:
Message should be a bit more "raw"
```suggestion
"""New favorites table added
```